### ISSUE TITLE
UserStreamListener:onDirectMessage() doesn't work

### ISSUE THREAD

**Issue by mustafacil** [Twitter4J/Twitter4J#321]
Hi. 
I can't listen the direct messages. I think it is because of new twitter api. Any update for this problem?

**Comment by ahmedeldeeb25** [comment]
@mustafacil  also video uploading not with me, Are they change the API?

**Comment by mustafacil** [comment]
@ahmedeldeeb25 yes. But the change is about stream api. Stream api is removed by twitter. The last issue is this topic: 
https://github.com/Twitter4J/Twitter4J/issues/292

Probably your problem may arise from removing the stream api.

**Comment by ahmedeldeeb25** [comment]
@mustafacil maybe, but it's a strange thing it was working with me just from a week but now not.

**Comment by 125569489** [comment]
You can try this https://twitter.utools.me/ tel: https://t.me/Yang0619
